Tasks/MSBuild/MSBuild.ps1

```diff
@@ -97,7 +97,7 @@ if ($cleanBuild)
 $nugetPath = Get-ToolPath -Name 'NuGet.exe'
 if (-not $nugetPath -and $nugetRestore)
 {
-    Write-Warning "Unable to locate nuget.exe. Package restore will not be performed for the solutions"
+    Write-Warning (Get-LocalizedString -Key "Unable to locate nuget.exe. Package restore will not be performed for the solutions")
```

Comment:
Don't forget to bump all the versions of the Tasks you changed in the Task.json and build using Gulp. I don't think you have to worry about minimunAgentVersion, but you can check with Ting.